### PR TITLE
feat(quota.increase-request): implement tracking when a user request a quota increase

### PR DESCRIPTION
**Pull request by brofik**
Signed-off-by: Rafik Adiche <rafik.adiche.ext@corp.ovh.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-9615` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-9615 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by anooparveti — DISMISSED** [review]
**suggestion:** In the file 'packages/manager/modules/pci/src/projects/project/quota/increase-request/increase-request.controller.js',
The common part of the tracking label `PublicCloud::pci::projects::project::quota`, can be maintained as constant.

**Review by anooparveti — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/quota/increase-request/increase-request.controller.js [hunk 1]
```diff
@@ -46,18 +48,66 @@ export default class PciProjectQuotaIncreaseController {
...
+  trackPopupCancel() {
+    if (
+      this.mode === QUOTA_INCREASE_MODES.BUY_CREDITS &&
+      this.serviceOptions.length > 0
+    ) {
+      return this.trackClick(
+        `${TRACK.BASE}::${TRACK.SELECT_PLAN}::${TRACK.CANCEL}`,
+      );
+    }
+    if (this.mode === QUOTA_INCREASE_MODES.CONTACT_SUPPORT) {
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/quota/increase-request/increase-request.controller.js:87]:
```suggestion
    else if (this.mode === QUOTA_INCREASE_MODES.CONTACT_SUPPORT) {
```

File: packages/manager/modules/pci/src/projects/project/quota/increase-request/increase-request.controller.js [hunk 2]
```diff
@@ -46,18 +48,66 @@ export default class PciProjectQuotaIncreaseController {
...
+  }
+
+  increaseQuota() {
+    if (
+      this.mode === QUOTA_INCREASE_MODES.BUY_CREDITS &&
+      this.serviceOptions.length > 0
+    ) {
+      return this.increaseQuotaByCredits();
+    }
+    if (this.mode === QUOTA_INCREASE_MODES.CONTACT_SUPPORT) {
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/quota/increase-request/increase-request.controller.js:102]:
```suggestion
    else if (this.mode === QUOTA_INCREASE_MODES.CONTACT_SUPPORT) {
```

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/quota/increase-request/increase-request.controller.js (hunk 1 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/quota/increase-request/increase-request.controller.js:87]: Inside the "if" statement is a "return" statement. So there is no need to add an "else" statement.

**Pull request merged**